build.gradle

```diff
@@ -24,7 +24,7 @@ plugins {
 
 
 allprojects {
-    group = 'org.ballerinalang'
+    group = 'io.ballerina'
```

Comment:
```suggestion
    group = project.group
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-jballerina.java.arrays/pull/195/commits/853a715b9e2b754ebddd75a2af46cde9780a6ead